src/kvstore/RocksEngineConfig.cpp

```diff
@@ -186,5 +194,27 @@ bool loadOptionsMap(std::unordered_map<std::string, std::string> &map, const std
     return true;
 }
 
+rocksdb::StatsLevel convertToStatsLevel(const std::string& str) {
+    if (str == "kExceptHistogramOrTimers") {
+        return rocksdb::StatsLevel::kExceptHistogramOrTimers;
+    } else if (str == "kExceptTimers") {
+        return rocksdb::StatsLevel::kExceptTimers;
+    } else if (str == "kExceptDetailedTimers") {
+        return rocksdb::StatsLevel::kExceptDetailedTimers;
+    } else if (str == "kExceptTimeForMutex") {
+        return rocksdb::StatsLevel::kExceptTimeForMutex;
+    } else {
+        return rocksdb::StatsLevel::kAll;
+    }
+}
+
+std::shared_ptr<rocksdb::Statistics> getDBStatistics() {
+    if (FLAGS_rocksdb_statistics_enable) {
+        static std::shared_ptr<rocksdb::Statistics> dbstats = rocksdb::CreateDBStatistics();
+        return dbstats;
+    }
+    return nullptr;
+}
+
```

Comment:
Suggest to put the creation and initialization of `rocksdb::Statistics`  into a single function.